### ISSUE TITLE
gdal installation issues

### ISSUE THREAD

**Issue by basaks** [GeoscienceAustralia/PyRate#148]
GDAL python bindings are available only for gdal versions: 1.5.0, 1.5.2, 1.6.0, 1.6.1, 1.7.0, 1.7.1, 1.8.1, 1.9.0, 1.9.1, 1.10.0, 1.11.0, 1.11.1, 1.11.2, 2.0.0, 2.0.1, 2.1.0, 2.1.3.

Make sure to install one of these versions.

Here is how to do it:

```
curl -O http://download.osgeo.org/gdal/2.1.0/gdal-2.1.0.tar.gz
tar -xzf gdal-2.1.0.tar.gz
cd gdal-2.1.0/
./configure --with-python
sudo make install
```

Should capture this in instructions somewhere.

**Comment by mcgarth** [comment]
Should be OK if PyPI, anaconda or virtualenv is used to sort the dependencies?

**Comment by basaks** [comment]
This relates to a `vitualenv`. Probably won't matter in `anaconda`.

**Comment by basaks** [comment]
Installation from pip (`pip install py-rate`) may still require
```pip install GDAL==$(gdal-config --version)```.

Installation from source works fine.

**Comment by brenmous** [comment]
There are now Python bindings up to GDAL 3.0.0. The versions we're targeting (2.2.2/2.2.3) have bindings and the advantage of 2.2.2 being an NCI module and 2.2.3 being available through apt on Ubuntu 18.04, so shouldn't have to build from source (this will likely change in the future as we support more platforms).
